### PR TITLE
MM-27212: Cypress/ee2: Add user to channel from popover

### PR DESCRIPTION
**Pull request by agnivade**
This PR automates MM-T3

https://mattermost.atlassian.net/browse/MM-27212

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

### CODE REVIEW

File: e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js [hunk 1]
```diff
@@ -67,4 +71,36 @@ describe('Profile popover', () => {
...
+        });
+
+        // Add to a Channel
+        cy.findByText('Add to a Channel').should('be.visible').click();
+
+        cy.get('div[aria-labelledby="addChannelModalLabel"]').within(() => {
+            // Type "private" and press enter.
+            cy.get('input').should('be.visible').type('private').wait(TIMEOUTS.HALF_SEC).type('{enter}');
+
+            // Verify that button is disabled
```
Comment (isacikgoz) [inline e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js:102]:
```suggestion
            // * Verify that button is disabled
```

File: e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js [hunk 2]
```diff
@@ -67,4 +71,36 @@ describe('Profile popover', () => {
...
+    it('MM-T2 Add user — Public ON / Private OFF', () => {
+        // Do the system console setup
+        cy.apiAdminLogin();
+        cy.visit('/admin_console/user_management/permissions/system_scheme');
+        cy.findByTestId('all_users-private_channel-checkbox').scrollIntoView().should('be.visible').click();
+        cy.findByTestId('saveSetting').should('be.visible').click();
+
+        cy.apiLogout();
+
+        // Login as the second user now
```
Comment (isacikgoz) [inline e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js:84]:
```suggestion
        // # Login as the second user now
```

**Review by isacikgoz — APPROVED** [review]
It looks good, just a note about the notation: `#` indicates a test step and `* `indicates an assertion like below. These should be added to the comments if necessary.

**Comment by agnivade** [comment]
Ahh, so that's the reason. :bulb: I was wondering why do we have those weird notations in the comments. I think I'll close this and send a single PR for the rest of the test cases combined. It's getting a bit cumbersome creating a long PR chain. And I don't want to be blocked until earlier PRs are reviewed.

**Comment by saturninoabril** [comment]
@agnivade The comments convention is intended for non-technical reviewer like QA analyst so it's easier to understand what's going on in the script (instead of trying to understand the code itself) and verify if it's similar to what was written in the TM4J (which is the source of truth for test management).

**Comment by agnivade** [comment]
Oh dear, I completely missed that. Thanks :)

**Pull request closed without merging**